OpenRA.Mods.Common/Traits/Render/WithCargoPipsDecoration.cs

```diff
@@ -69,7 +69,6 @@ string GetPipSequence(int i)
 				{
 					var sequence = Info.FullSequence;
 					if (pi.CustomPipType != null && !Info.CustomPipSequences.TryGetValue(pi.CustomPipType, out sequence))
-						Log.Write("debug", "Actor type {0} defines a custom pip type {1} that is not defined for actor type {2}".F(c.Info.Name, pi.CustomPipType, self.Info.Name));
```

Comment:
You forgot to undo to removing this as this if statement seems to lead nowhere?

Reply:
Derp. Well, removing this was intentional, but of course the if can't stay like that, good catch.